src/js/media-play-button.ts

```diff
@@ -28,13 +32,43 @@ slotTemplate.innerHTML = /*html*/ `
     <slot name="play">${playIcon}</slot>
     <slot name="pause">${pauseIcon}</slot>
   </slot>
+  <media-tooltip id="tooltip">
```

Comment:
I'm wondering if we want something like the [slot template](https://github.com/muxinc/media-chrome/blob/67eab2311a000bbd2a7dbf8be97b43cc392f0d29/src/js/media-play-button.ts#L15) we use for icons here.

Then every button subclass wouldn't have to bring its own tool tip, it'd just have to define the contents of the tooltip. And also the whole tool tip could be replaced with a custom one.